pkg/util/docker/event_stream.go

```diff
@@ -16,138 +16,106 @@ import (
 	log "github.com/cihub/seelog"
 	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/api/types/filters"
-
-	"github.com/DataDog/datadog-agent/pkg/status/health"
```

Comment:
As this dispatch goroutine is now blocking, removing the heathcheck as it would be non-trivial, and would not give more insights (it's dependant on the user goroutine to be healthy)